src/components/InputItem/InputItem.module.css

```diff
@@ -0,0 +1,3 @@
+.wrap {
+    padding-bottom: 15px;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5693557)